src/api/hooks/useBalance.ts

```diff
@@ -0,0 +1,27 @@
+import {AccountInfo} from '@polkadot/types/interfaces';
+import {useApi} from 'context/ChainApiContext';
+import {useEffect, useState} from 'react';
+
+export function useBalance({address}: {address: string}) {
+  const {api} = useApi();
+  const [balance, setBalance] = useState<AccountInfo | null>(null);
+
+  useEffect(() => {
+    let localUnsub: () => void | null;
+    if (api) {
+      api?.query.system
+        .account(address, (accountInfo) => {
```

Comment:
We already have a hook for this (`useAccountInfo`)

Reply:
thanks 👍  I added it on 843ce24  ✅